stdlib/public/TensorFlow/NumpyConversion.swift

```diff
@@ -109,6 +109,22 @@ extension Tensor : ConvertibleFromNumpyArray
   }
 }
 
+extension ShapedArray where Scalar : NumpyScalarCompatible {
+  // Creates a `ShapedArray` instance from the given `PythonObject` if it is a
+  // `numpy.ndarray` instance with a matching scalar datatype.
+  public init?(_ pythonObject: PythonObject) {
+    self.init(numpyArray: pythonObject)
```

Comment:
If it's not a numpy array, we should load it as a (potentially multi-dimensional) python array.

Reply:
Good point. Added TODO comments in https://github.com/apple/swift/pull/22104/commits/49f70e4e5ee106f45493308a2ce308929e2d20ac.